docs/_plugins/markers-renderers/leaflet-coloricon.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Leaflet.ColorIcon
+category: markers-renderers
+repo: https://github.com/shevekk/Leaflet.ColorIcon
+author: Maxence Martin (shevek)
+author-url: https://github.com/shevekk
+demo: http://dataexplorer.hd.free.fr/Leaflet.ColorIcon/examples/basic/
+compatible-v0:
```

Comment:
Is your plugin created for V0? I think not but please confirm with changing to `compatible-v0: false`